### ISSUE TITLE
docs out of sync: 'none' is not allowed for 'log-level-file'

### ISSUE THREAD

**Issue by c4milo** [pgbackrest/pgbackrest#574]
I was trying to disable logging to files since I run pgbackrest inside a container that logs all to stdout and stderr. However, I bumped into this inconvenience:

Docs at https://pgbackrest.org/configuration.html#section-repository/option-log-path say the following:
```
The log path provides a location for pgBackRest to store log files. Note that if log-level-file=none then no log path is required.
```
Setting `log-level-file=none` renders the following error: 
```
ERROR: [032]: 'none' is not allowed for 'log-level-file' option
```

Being able to disable file logging by setting `log-level-file=none` is what I was looking for. I hope it can be reintroduced. Thanks!

**Comment by c4milo** [comment]
Ah, I finally understood the logging options. Setting `log-level-file=off` seems to be it!

**Comment by dwsteele** [comment]
Definitely a typo.  Fixed at 40924e8f5c84cfd.
